crates/bevy_ui/src/ui_node.rs

```diff
@@ -412,26 +418,26 @@ impl Default for AlignSelf {
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect(PartialEq, Serialize, Deserialize)]
 pub enum AlignContent {
-    /// Items are packed toward the start of the axis
+    /// Each line moves towards the start of the cross axis.
```

Comment:
"cross axis" is true for Flexbox. But Grid has no "main" or "cross" axis. It only has "block" (vertical with left-to-right writing mode) and "inline" (horizontal with left-to-right writing mode). For grid, the `align*` properties apply to the block axis and the `justify*` properties apply to the inline axis.